core/src/main/java/tech/tablesaw/columns/numbers/DoubleParser.java

```diff
@@ -24,9 +26,14 @@ public boolean canParse(String s) {
       return true;
     }
     try {
-      Double.parseDouble(AbstractColumnParser.remove(s, ','));
+      if (isPercent(AbstractColumnParser.remove(s, ','))) {
+        s = AbstractColumnParser.remove(s, ',');
+        Number number = NumberFormat.getPercentInstance().parse(s);
```

Comment:
remove unused variable
```suggestion
        NumberFormat.getPercentInstance().parse(s);
```